common/lib/container-definitions/src/loader.ts

```diff
@@ -334,7 +334,7 @@ export interface ILoaderHeader {
     [LoaderHeader.loadMode]: IContainerLoadMode;
     [LoaderHeader.sequenceNumber]: number;
     [LoaderHeader.reconnect]: boolean;
-    [LoaderHeader.version]: string | undefined;
+    [LoaderHeader.version]: string | undefined | null;
```

Comment:
@jatgarg is this right? or should it be removed in main?

Reply:
I believe null was deprecated in main.
Here, similar to logging change, the direction should be reverse - find a way to make these changes in 0.39, or bump definitions up (to 0.40) as they are both breaking changes.

Reply:
yup. that sounds right. i'm looking into what bumping to 0.40 looks like, so just reviewing changes. pr seemed the best way to get feedback/answers to understand the breaks.